src/main/webapp/myprofile.html

```diff
@@ -31,19 +32,43 @@ <h1>User Directory</h1>
       <div class="wrapper">
         <!-- User Details -->
         <div class="details-container">
-            <img src="/images/avatar.jpg"></img>
+            <img class="profilepic" src="/images/avatar.jpg"></img>
             <h2>Name</h2>
             <h3>Department</h3>
             <p>Hello I am interested in __ please feel free to contact me for __</p>
-            <button class="button">Change my profile</p>
+            <button class="button" onclick="togglePopup()">Edit profile</p>
         </div>
         <!-- User's Recommendation Map -->
-        <div id="map"></div>  
+        <div class="map-container">
+          <h2>My favorite places</h2>
+          <div id="map"></div>  
+        </div>
+        <!-- Pop up box to change profile -->
+        <div id="popup-profile">
+          <div class="overlay"></div>
+          <div class="popup-content">
+            <div class="close-button" onclick="togglePopup()">&times;</div>
+            <h2>Edit Profile</h2>
+            <p>Please change your profile by filling up this form and press submit</p>
+            <!-- Form for changing profile -->
+            <form>
+              <label for="Name">Full name:</label><br>
+              <textarea type="text" id="name" name="name" cols="40" rows="1" required></textarea><br>
+              <label for="Department">Department:</label><br>
+              <textarea type="text" id="department" name="department" cols="40" rows="1" required></textarea><br>
+              <label for="Bio">Bio:</label><br>
+              <textarea type="text" id="bio" name="bio" cols="40" rows="3" required></textarea><br>
+              <!-- <label for="Profile Picture">Profile picture:</label><br>
```

Comment:
Rather than commenting out this, shall we delete, since we are not using it?

Reply:
we are going to use it to allow changing profile picture, but I was not sure yet since this will need more research involving image upload